modules/mlops/kubeflow-users/kustomize/config.template

```diff
@@ -0,0 +1,23 @@
+issuer: http://dex.auth.svc.cluster.local:5556/dex
+storage:
+  type: kubernetes
+  config:
+    inCluster: true
+web:
+  http: 0.0.0.0:5556
+logger:
+  level: "debug"
+  format: text
+oauth2:
+  skipApprovalScreen: true
+enablePasswordDB: true
+staticPasswords:
+- email: user@example.com
+  hash: $2y$12$4K/VkmDd1q1Orb3xAt82zu8gk7Ad6ReFR4LCP9UeYE90NLiN9Df72
```

Comment:
is this file checked in intentionally? Do we have any risk here?

Reply:
This is a one-way hash that cannot be deciphered....it is also an example of a user that does not exist.  There is no rsk here

Reply:
Same for `userID` in line 18? 
No meaning on this sequence of this digits?
Just in case.

Reply:
Correct